pkgs/by-name/pr/prisma-engines_6/package.nix

```
@@ -12,14 +11,14 @@
 # version of prisma-engines and prisma must be the same for them to
 # function correctly.
 rustPlatform.buildRustPackage rec {
-  pname = "prisma-engines";
-  version = "6.19.0";
+  pname = "prisma-engines_6";
+  version = "6.19.1";
 
   src = fetchFromGitHub {
     owner = "prisma";
     repo = "prisma-engines";
     rev = version;
```

```suggestion
    tag = version;
```

Nit, but since we're changing a bunch of stuff might as well.

Ditto for all other packages (and if you don't mind, do the same in the prisma overrides in the umami package)